calendar-bundle/contao/classes/Events.php

```diff
@@ -493,6 +493,11 @@ public static function getSchemaOrgData(CalendarEventsModel $objEvent): array
 			'startDate' => $objEvent->addTime ? date('Y-m-d\TH:i:sP', $objEvent->startTime) : date('Y-m-d', $objEvent->startTime)
 		);
 
+		if ($objEvent->endDate)
+		{
+			$jsonLd['endDate'] = $objEvent->addTime ? date('Y-m-d\TH:i:sP', $objEvent->endTime) : date('Y-m-d', $objEvent->endTime);
```

Comment:
According to schema.org, [this should always be ISO_8601](https://schema.org/endDate). So imho we need to create a `DateTimeImmutable` instance here and format according to `\DateTimeInterface::ISO8601`.

Reply:
The line is an exact copy of line 493. Is it wrong there, too?

Reply:
I think we should change both then, yes.

Reply:
Let‘s create a separate PR for that then, because we use `'Y-m-d\TH:i:sP'` at several places in the codebase and we should change all of them or none.

Reply:
See https://github.com/contao/contao/issues/6500.